frontend/components/TableContainer/DataTable/TooltipTruncatedTextCell/TooltipTruncatedTextCell.tsx

```diff
@@ -15,44 +15,30 @@ interface ITruncatedTextCellProps {
   classes?: string;
 }
 
-const baseClass = "truncated-cell";
+const baseClass = "tooltip-truncated-cell";
 
-const TruncatedTextCell = ({
+const TooltipTruncatedTextCell = ({
   value,
   tooltipBreakOnWord = false,
   classes = "w250",
-}: ITruncatedTextCellProps): JSX.Element => {
+}: ITooltipTruncatedTextCellProps): JSX.Element => {
   const classNames = classnames(baseClass, classes, {
     "tooltip-break-on-word": tooltipBreakOnWord,
   });
 
-  const ref = useRef<HTMLInputElement>(null);
-
-  const [offsetWidth, setOffsetWidth] = useState(0);
-  const [scrollWidth, setScrollWidth] = useState(0);
-
-  useLayoutEffect(() => {
-    if (ref?.current !== null) {
-      setOffsetWidth(ref.current.offsetWidth);
-      setScrollWidth(ref.current.scrollWidth);
-    }
-  }, []);
-
```

Comment:
Is this being removed because of the general tooltip refactor?

Reply:
exactly